extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -385,6 +387,9 @@ export const DefaultCollationProperty = 'DefaultCollation';
 /** Default database collation to use when none is specified in the project */
 export const DefaultCollation = 'SQL_Latin1_General_CP1_CI_AS';
 
+/** Well-known values that are allowed to be sent in telemetry */
+export const WellKnownValues = ['dsct-oracle-to-ms-sql'];
```

Comment:
For both, javadoc and the name itself, should we rename this to something more specific, like `WellKnownDatabaseSources`?